account_credit_control_dunning_fees/model/line.py

```diff
@@ -13,7 +13,8 @@ class CreditControlLine(models.Model):
     balance_due_total = fields.Float(string='Balance due with fees',
                                      compute='compute_balance_due')
 
-    @api.one
+    @api.multi
     @api.depends('dunning_fees_amount', 'balance_due')
     def compute_balance_due(self):
+        self.ensure_one()
```

Comment:
It is true that sometimes, you can replace `@api.one` with `@api.multi` + `self.ensure_one()`, but then you need to make sure that the method is not called with more than one record. In the cases in this PR, this is not the case. For instance, this is a compute method from a field definition, and those are commonly called on multiple records. So you will need to change these methods to loop over `self` if you change the decorator.

Reply:
The @api.one decorator is a way to have the ORM do the looping. Such a method can be called on multiple records.